### PR TITLE
Add stability warning to README & book (ref #117)

### PR DESCRIPTION
**Pull request by phated**
@mwilliammyers How do you feel about this warning? Definitely looking for ways to improve it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,16 @@
 * [Cargo package](https://crates.io/crates/async-graphql)
 * Minimum supported Rust version: 1.42 or later
 
+## Stability: Unstable & Experimental
+
+__This project doesn't currently follow SemVer, and there can be breaking changes on any version numbers. We will begin following SemVer once the project reaches v2.0.0__
+
+Even though this project is above v1.0.0, we are rapidly changing and improving the API. This has caused versioning problems that aren't easily resolved because the project became popular very quickly (it was only started in March 2020).
```
Comment (mwilliammyers) [inline README.md:39]: Should we use code style for version strings? e.g.`v1.0.0` ?

File: README.md [hunk 2]
```diff
@@ -32,6 +32,16 @@
...
+
+We currently plan to start following SemVer once we reach the v2.0.0 release, which will happen once the API starts to stabilize. Unfortunately, we don't currently have the timeline for this.
+
+If things don't seem to be compiling after an upgrade, it is likely you'll need to dive into compiler errors to update some syntax that changed. Feel free to open an issue if something seems weird!
```
Comment (mwilliammyers) [inline README.md:43]: Should we add something like?

> In accordance with Rust's policy on pre-`1.0.0` crates, we will try to keep breaking changes limited to `minor` version changes, but if things don't seem to be ...

Comment (mwilliammyers) [inline README.md:43]: Should we add a link to the issues?

File: docs/en/src/introduction.md [hunk 3]
```diff
@@ -8,10 +8,15 @@ You can define a Schema in Rust and procedural macros will automatically generat
 
 I like GraphQL and Rust. I've been using `Juniper`, which solves the problem of implementing a GraphQL server with Rust. But Juniper had several problems, the most important of which is that it didn't support async/await at the time. So I decided to make this library for myself.
 
-## Progress
+## Stability: Unstable & Experimental
```
Comment (mwilliammyers) [inline docs/en/src/introduction.md:11]: Same notes as above :)

File: README.md [hunk 4]
```diff
@@ -32,6 +32,16 @@
 * [Cargo package](https://crates.io/crates/async-graphql)
 * Minimum supported Rust version: 1.42 or later
 
+## Stability: Unstable & Experimental
+
+__This project doesn't currently follow SemVer, and there can be breaking changes on any version numbers. We will begin following SemVer once the project reaches v2.0.0__
```
[the remark this answers is not in the document]
Comment (mwilliammyers) [reply README.md:37]: If that is too wordy, we should definitely link to the spec.

**Comment by sunli829** [comment]
Thank you. 😂

**Comment by phated** [comment]
Sorry, I didn't get the suggestions added before this got merged. I will get those addressed in another PR.
